tests/admin_changelist/models.py

```diff
@@ -78,7 +78,11 @@ class Swallow(models.Model):
     uuid = models.UUIDField(primary_key=True, default=uuid.uuid4)
     origin = models.CharField(max_length=255)
     load = models.FloatField()
-    speed = models.FloatField()
+    speed = models.FloatField(help_text='Speed')
+
+    def model_method(self):
+        return 'model method'
+    model_method.help_text = 'Model Method'
```

Comment:
Can we add tests for properties?

Reply:
Sorry, Couldn't figure out what you meant. Could you please explain it?

Reply:
Elements of list_display can also be properties. [See](https://docs.djangoproject.com/en/3.0/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_display) how we handle `short_description`.

Reply:
Thanks @felixxm. I added it.